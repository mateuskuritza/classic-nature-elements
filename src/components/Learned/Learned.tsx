import styled from "styled-components";

export default function Learned() {
	return (
		<Container>
			<p>Dominado!</p>
		</Container>
	);
}

const Container = styled.div`
	background-color: green;
	color: white;
	width: 90%;
	text-align: center;
	padding: 2px;
	border-radius: 4px;
`;
