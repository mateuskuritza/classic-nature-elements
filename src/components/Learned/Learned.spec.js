import React from "react";
import { render, screen } from "@testing-library/react";
import Learned from "./Learned";

it("Should render Learned", () => {
	const { container } = render(<Learned />);
	const button = screen.getByText("Dominado!");
	expect(button).toBeDefined();
	expect(container).toMatchSnapshot();
});
