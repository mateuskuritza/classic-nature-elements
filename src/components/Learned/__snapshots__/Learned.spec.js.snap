// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Learned 1`] = `
<div>
  <div
    class="sc-bdvvtL IjgTa"
  >
    <p>
      Dominado!
    </p>
  </div>
</div>
`;
